### DISCUSSION TITLE
Multiple Business Cards Per User

### DISCUSSION THREAD

**Discussion by Trent-Daniel** [Trent-Daniel/DigitalBusinessCard#7] [RFI]
We need to discuss how this will work. Ideas:
- the user save a series of templates to their profile and then select one ahead of time to be the business card that others see
- the user saves a series of templates to their profile and assigns tags to each. Different viewers would see the business card with complimentary tags. E.g. if a person tagged one business card with #medical and another with #cooking, a viewer with an account tagged #medical would view the card tagged with #medical instead of the one tagged with #business. This would be helpful if we implemented the idea of having certifications posted, and if users wanted only certain certifications posted for certain viewers
- the user could have different business cards for different interacting with different cultures or in different languages

All other ideas are welcome!

**Comment by DanielPageau** [comment]
- Each user should have a DBC create page where they can create different cards with different names
- The cards should have unique URL's that can be shared and found
